sdk/core/System.ClientModel/src/Internal/ReadTimeoutStream.cs

```diff
@@ -60,26 +59,26 @@ public override void Flush()
 
     public override int Read(byte[] buffer, int offset, int count)
     {
-        var source = StartTimeout(default, out bool dispose);
+        CancellationTokenSource source = StartTimeout(default, out bool dispose);
```

Comment:
```suggestion
        using CancellationTokenSource source = StartTimeout(default, out bool dispose);
```

Reply:
This is existing Azure.Core code - let me see if handling this correctly causes any problems!

Reply:
Tracking with https://github.com/Azure/azure-sdk-for-net/issues/41932